### ISSUE TITLE
New user does not have access to the entire dashboard history

### ISSUE THREAD

**Issue by ZerooCool** [go-gitea/gitea#9643]
First install on linux Debian Buster
cd /usr/local/bin/
sudo wget -O /usr/local/bin/gitea https://dl.gitea.io/gitea/1.9.4/gitea-1.9.4-linux-amd64

Then Ugrade : 
sudo wget -O /usr/local/bin/gitea https://dl.gitea.io/gitea/1.10.0-rc1/gitea-1.10.0-rc1-linux-amd64
And : 
sudo wget -O /usr/local/bin/gitea https://dl.gitea.io/gitea/1.10.2/gitea-1.10.2-linux-amd64

Gitea : 1.10.2 built with GNU Make 4.1, go1.13.5 : bindata, sqlite, sqlite_unlock_notify
Git : 2.20.1

  - [ ] MSSQL -> I use MariaDB

- Can you reproduce the bug at https://try.gitea.io:
  - [ ] No because i'm not admin, i can't create : 
First a Organisation, 
Then add a dépot to the Organisation,
Then create a group and add user, make commits ..

Now, create a new user, and, add the new user to organisation team.
The new user can't see the old history from dashboard

If a new commit income, new user can see the new commit from dashboard

My tutorial for install Gitea (in french) :
https://wiki.visionduweb.fr/index.php?title=Installer_un_serveur_Git_avec_Gitea#Installer_Gitea_depuis_les_sources

I modified the template as follows:
https://wiki.visionduweb.fr/index.php?title=Modifier_le_template_de_Gitea

Step for log : 
https://docs.gitea.io/en-us/logging-configuration/
I did not understand well, nor advanced in this direction. The logs therefore remain obscure. If necessary, please accompany me to obtain the desired logs.

**Comment by 6543** [comment]
.. add user to organisation team

**Comment by guillep2k** [comment]
>
>
> Now, create a new user, and, add the new user to organisation team.
> The new user can't see the old history from dashboard

The dashboard is for user notifications, not for a project's history. It's normal that a user starts with no notifications.

**Comment by ZerooCool** [comment]
Ok, in use, it can be understood as well.
I had felt it, for that I had tested to push two new commit, which were well visible.

On the other hand, it is not so obvious to note, once we have a few users who are all well notified, on the dashboard, and, that a new user, him, arrives with an empty space.

I spent a lot of time figuring out how to update the information for the new user, thinking, along the lines of other users, that I could do something.

So it's not a bug, so much the better.
It seems "obvious" now.

I remain on my hunger by thinking that I could have allowed this new user to get the history from his dashboard.
